components/core/src/clp/clo/CommandLineArguments.cpp

```diff
@@ -219,21 +232,37 @@ CommandLineArguments::parse_arguments(int argc, char const* argv[]) {
 
             print_basic_usage();
             cerr << "OUTPUT_HANDLER is one of:" << endl;
-            cerr << "  " << static_cast<char const*>(cResultsCacheOutputHandlerName)
-                 << " - Output to the results cache" << endl;
-            cerr << "  " << static_cast<char const*>(cReducerOutputHandlerName)
-                 << " - Output to the reducer" << endl;
+            cerr << "  " << cNetworkOutputHandlerName << " - Output to a network destination"
```

Comment:
Why remove the static cast?

Reply:
Do we need to explicitly cast the variable?

Reply:
Yeah, according to this clang-tidy [warning](https://clang.llvm.org/extra/clang-tidy/checks/cppcoreguidelines/pro-bounds-array-to-pointer-decay.html), we should explicit when we want to decay an array into a pointer (clang-tidy says to cast it, but the docs don't for some reason).